### PR TITLE
Add benchmarks for paper

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

File: healpix_alchemy/tests/benchmarks/data.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        monkeypatch.setattr(np, 'random', np.random.default_rng(seed))
+        return uniform_spherical_random_surface(n)
+
+
+def get_random_galaxies(n, cursor):
+    points = SkyCoord(get_random_points(n, RANDOM_GALAXIES_SEED))
+    hpx = HPX.skycoord_to_healpix(points)
+
+    f = io.StringIO('\n'.join(f'{i}' for i in hpx))
+    cursor.copy_from(f, Galaxy.__tablename__, columns=('hpx',))
```
Comment (mcoughlin) [inline healpix_alchemy/tests/benchmarks/data.py:70]: Can you explain what is going on with the cursor stuff?

**Comment by mcoughlin** [comment]
@lpsinger Maybe should also add the command line to the README to trigger the tests locally?

File: healpix_alchemy/tests/benchmarks/data.py (hunk 1 above)
Comment (lpsinger) [reply healpix_alchemy/tests/benchmarks/data.py:70]: Yeah, this is much faster than using SQL. See the docstring.

**Comment by lpsinger** [comment]
> @lpsinger Maybe should also add the command line to the README to trigger the tests locally?

I'll take care of that in a separate issue, #35.

**Comment by mcoughlin** [comment]
With the addition of the other PR, looks great to me.

**Review by mcoughlin — APPROVED** [review]

**Pull request merged**